src/pages/resources/hackathon-resources.mdx

```diff
@@ -41,12 +31,27 @@ Always refer back to the [getting help](/resources/getting-help) page for the la
 - [Hackathon Discord](https://discord.gg/h3AvTHj)
 - [Developer Discord](https://discord.gg/2YHSAey)
 
-## Tutorials
+## Documentation and tutorials
+
+If you are new to Smart Contracts, read the [Getting Started Guide](/getting-started/conceptual-overview/).
 
-Be sure to check the documentation in the links above for in depth-descriptions of everything.
+### Service documentation
 
-New? See the [Getting Started Guide](/getting-started/conceptual-overview/).
+- [Data Feeds](/data-feeds)
```

Comment:
This section may be redundant since the page already lives in the documentation. I'm leaning toward removing it. Thoughts?

Reply:
Done.